exporters/otlp/otlpmetric/header.go

```diff
@@ -12,13 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package internal contains common functionality for all OTLP exporters.
-package internal // import "go.opentelemetry.io/otel/exporters/otlp/internal"
+package otlpmetric // import "go.opentelemetry.io/otel/exporters/otlp/otlpmetric"
 
-import "go.opentelemetry.io/otel"
-
-// GetUserAgentHeader return an OTLP header value form "OTel OTLP Exporter Go/{{ .Version }}"
+// GetUserAgentHeader returns an OTLP header value form "OTel OTLP Exporter Go/{{ .Version }}"
 // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#user-agent
 func GetUserAgentHeader() string {
```

Comment:
This function needs to be moved to a local internal directory or be unexported. Otherwise this change will make it a part of the public API.

Similarly for the trace exporter.